jwql/utils/constants.py

```diff
@@ -205,6 +205,18 @@
 # Types of potential bad pixels identified by the dark current monitor
 DARK_MONITOR_BADPIX_TYPES = ["hot", "dead", "noisy"]
 
+# Minimum amount of time, in days, between epochs of dark current observations. If the
+# dark monitor sees this much time, or longer, between two dark current files, it assumes
+# that the two files are part of separate epochs. This means the monitor will run separately
+# on these files, rather than bundling them together into a batch, where they would have
+# been combined into a mean dark rate
+DARK_MONITOR_BETWEEN_EPOCH_THRESHOLD_TIME = {'nircam': 10.,
+                                             'niriss': 10.,
+                                             'miri': 0.00001,  # Treat each MIRI exposure separately
```

Comment:
Just looked this up, MIRI epochs are `0.864` seconds apart 😆

Reply:
MIRI said that each file should always be considered on its own. No averaging multiple files. This seemed like the easiest way to do that. For NIRCam, we only take one exposure in each observation, so the time here doesn't really matter much. It'll be one file at a time there as well.

Reply:
Definitely makes sense, I was just curious what the time in seconds would be so I am sure we will catch each individual file with this rate!